wolfcrypt/src/coding.c

```diff
@@ -136,9 +136,12 @@ static WC_INLINE int Base64_SkipNewline(const byte* in, word32 *inLen, word32 *o
             WOLFSSL_MSG("Bad end of line in Base64 Decode");
             return ASN_INPUT_E;
         }
-        curChar = in[j];
+
+        if (len) {
+            curChar = in[j];
+        }
     }
-    while (len && curChar == ' ') {
+    while (len > 1 && curChar == ' ') {
```

Comment:
Still want Len to decrement to 0 when curChar is a space.
Check len is not 0 in loop before getting new curChat.

Reply:
Not sure what you mean, can you please paste the desired code block?

Reply:
while (len && curChar == ' ') {
    if ((--len) > 0)
        curChar = in[++j];
}

This way, the length goes to zero and an error is returned.

Reply:
Done. Thanks for the suggestion. I left out the extra parentheses for consistency with the ```if (---var ...)``` pattern elsewhere in the code.

Reply:
Looks good to me! :-)